examples/access-control-migration/README.md

```diff
@@ -1,6 +1,306 @@
 # @loopback/example-access-control-migration
 
-This is the basic tutorial for getting started with Loopback 4!
+This example is migrated from
+[loopback-example-access-control](https://github.com/strongloop/loopback-example-access-control),
+and uses the authentication and authorization system in LoopBack 4 to implement
+the access control.
+
+# App Scenario
+
+This example is a donation system. Users can view or make donations to projects
+based on their roles. A project is owned by a user, and a user can create a team
+to involve other users as team members. The system has an administration.
+
+Here is the diagram that describes the models:
+
+![models](./imgs/scenario-model.png)
+
+And the Project endpoints with their allowed roles:
+
+- /Projects/list-projects
+  - everyone
+- /Projects/view-all
+  - admin
+- /Projects/{id}/show-balance
+  - owner, teamMember
+- /Projects/{id}/donate
+  - admin, owner, teamMember
+- /Projects/{id}/withdraw
+  - owner
+
+The diagram for models also include the pre-created data in this application.
+
+For example, u1 owns project1, it also creates a team with u1 and u2 as members.
+Which means u1 is the **owner** of project1, and u1 and u2 are the **team
+members** of project1. And u3 is the **admin**.
+
+# Difference
+
+LoopBack 3 has several built-in models that consists of a RBAC system. The
+models are User, Role, RoleMapping, ACL, AccessToken. You can learn how they
+work together in tutorial
+[Controlling data access](https://loopback.io/doc/en/lb3/Controlling-data-access.html).
+
+LoopBack 4 authorization system gives developers the flexibility to implement
+the RBAC on their own. You can leverage popular 3rd-party libraries like
+[casbin](https://github.com/casbin/casbin) or [oauth0](https://auth0.com/) for
+the role mapping.
+
+This guide includes a demo for using [casbin](#steps-with-casbin) and using
+[oauth0](#steps-with-oauth0)(TBD)
+
+# Steps with Casbin
```

Comment:
I found the title is not that accurate. IIUC, this doc focuses on _migrate_ ACL from LB3 to LB4. Maybe change it to something like `Migrate ACL to LB4 with Casbin`?

Reply:
reworded it to `Migrating Example to LB4 with Casbin`